integration/SimpleCacheClient.test.ts

```diff
@@ -12,36 +11,35 @@ if (!AUTH_TOKEN) {
 }
 const INTEGRATION_TEST_CACHE_NAME = process.env.TEST_CACHE_NAME || 'dummy';
 
-// TODO: deprecating credentials file for now
-// const momentoDirName = `${os.homedir()}/.momento`;
-// const credsFilePath = `${momentoDirName}/credentials.toml`;
-// const createSystemCredentials = (profile?: string) => {
-//   const profileName = profile ?? 'default';
-//   if (profile) {
-//     process.env.MOMENTO_PROFILE = profileName;
-//   }
-//   if (!fs.existsSync(momentoDirName)) {
-//     fs.mkdirSync(momentoDirName);
-//   } else {
-//     throw new Error(`${momentoDirName} directory exists.
-// These integration tests test reading profiles from disk, and create a ~/.momento directory to test this.
-// To avoid overriding existing profiles, this error has been thrown.
-// If you a want to run these tests, run "mv ~/.momento ~/.momento.bac" to save the current profiles.
-// After these tests complete run "mv ~/.momento.bac ~/.momento" to restore the profiles`);
-//   }
-//   fs.writeFileSync(
-//     credsFilePath,
-//     `[profile.${profileName}]
-// token = "${AUTH_TOKEN}"`
-//   );
-// };
-//
-// const removeSystemCredentials = () => {
-//   fs.rmSync(momentoDirName, {
-//     force: true,
-//     recursive: true,
-//   });
-// };
+const momentoDirName = `${os.homedir()}/.momento`;
```

Comment:
Just a thought but for tests we could write these to different location just to avoid wiping out local devs creds when running? Like 
```
const momentoDirName = `${os.homedir()}/.momento-test`;
```